### DISCUSSION TITLE
Error installing Mondoo Client

### DISCUSSION THREAD

**Discussion by misterpantz** [mondoohq/cnquery#806] [Q&A] [answered]
When I try to install Mondoo Client (as instructed in the Amazon AWS Setup wizard) I get this error:

`The Mondoo Client install script encountered a problem. For assistance, please join our community on GitHub Discussions.`

I'm in stalling on macOS. I have cnquery and cnspec already installed.

**Comment by benr** [comment]
I'll follow up with you by DM for some additional details!

**Comment by benr** [comment, marked as answer]
You found an edge case!  The problem is that you have Homebrew installed but installed Mondoo using a downloaded package (ie: https://releases.mondoo.com/mondoo/).  The script assumes you must have used Homebrew to install and tries to upgrade but fails.

I've created an Issue to track and resolve this problem. https://github.com/mondoohq/installer/issues/249

Thank you for reporting this!

**Comment by misterpantz** [comment]
Thanks, @benr!
